pkg/controllers/disruption/orchestration/queue.go

```diff
@@ -274,10 +271,9 @@ func (q *Queue) waitOrTerminate(ctx context.Context, cmd *Command) error {
 		if err := q.kubeClient.Delete(ctx, candidate.NodeClaim); err != nil {
 			multiErr = multierr.Append(multiErr, client.IgnoreNotFound(err))
 		} else {
-			metrics.NodeClaimsTerminatedCounter.With(prometheus.Labels{
-				metrics.ReasonLabel:       cmd.method,
-				metrics.NodePoolLabel:     cmd.candidates[i].NodeClaim.Labels[v1.NodePoolLabelKey],
-				metrics.CapacityTypeLabel: metrics.GetLabelOrDefault(cmd.candidates[i].Node.Labels, v1.CapacityTypeLabelKey),
+			metrics.NodeClaimsDisruptedTotal.With(prometheus.Labels{
+				metrics.ReasonLabel:   cmd.method,
```

Comment:
```suggestion
				metrics.ReasonLabel:   cmd.reason,
```

Change the data field name from `method` to `reason`?